external-crates/move/crates/move-analyzer/tests/ide_testsuite.rs

```diff
@@ -381,6 +381,7 @@ fn initial_symbols(
         pkg_deps.clone(),
         ide_files_root.clone(),
         project_path.as_path(),
+        None,
         LintLevel::None,
     )?;
 
```

Comment:
Can we add a test with some precompiled info just for sanity checking?

Reply:
We need a full compilation to compute all symbols and this is the first and only one that's done per test suite (as files do not change). What I did, though, is to add another partial compilation right after this one and use its results to run tests, but it lengthens the time to execute the test suite (not to unreasonable level, though)